examples/habitat_cl_example.py

```diff
@@ -0,0 +1,90 @@
+from argparse import ArgumentParser
```

Comment:
May want to add this script to tests/test_examples.py so it's run during code refactors.

Reply:
great point thanks